schemas/bedbase/output_fairtrack.yaml

```diff
@@ -0,0 +1,119 @@
+description: bedboss fairtrack output schema
+
+properties:
+  samples:
+    type: array
+    items:
+      type: object
+      properties:
+        local_id:
+          type: string
+          description: "Name of the bed sample (digest)"
+        global_id:
+          type: string
+          description: "Unique sample identifier of the bed source (e.g. 'encode:ENCBS192PUU', 'geo:GSM1234')"
+        species_id:
+          type: string
+          description: "Taxonomy id based on NCBI database (e.g. taxonomy:9606)"
+        species_name:
+          type: string
+          description: "organism name. (e.g. homo sapiens)"
+        biospecimen_class:
+          type: stirng
+          description: ????
```

Comment:
Examples of biospecimen class can be found here:
https://raw.githubusercontent.com/fairtracks/fairtracks_standard/master/json/blueprint/blueprint_augmented.json

Looks like it can be labels such as:
"cell", "organism_part"

Reply:
also seeing `Abnormal Cell`